test.js

```diff
@@ -294,7 +294,7 @@ test('!in, $type', function(t) {
 
 test('any', function(t) {
     var f1 = filter(['any']);
-    t.equal(f1({properties: {foo: 1}}), false);
+    t.equal(f1({properties: {foo: 1}}), true);
```

Comment:
This shouldn't change; returning `false` for an empty `any` is standard mathematical / functional programming behavior.